.devcontainer/devcontainer.json

```
@@ -0,0 +1,26 @@
+{
+	"name": " Codespace",
+	"image": "mcr.microsoft.com/vscode/devcontainers/javascript-node:0.202.3-16",
+	"settings": {
+		"workbench.colorTheme": "Default Dark+"
+	},
+	"extensions": [
+		"eamodio.gitlens",
+		"cschleiden.vscode-github-actions",
+		"redhat.vscode-yaml",
+		"bierner.markdown-preview-github-styles",
+		"jmrog.vscode-nuget-package-manager",
+		"coenraads.bracket-pair-colorizer",
+		"vscode-icons-team.vscode-icons",
+		"editorconfig.editorconfig",
+		"dbaeumer.vscode-eslint",
+		"ms-vscode.vscode-typescript-next",
+		"ecmel.vscode-html-css",
+		"streetsidesoftware.code-spell-checker",
+		"alexkrechik.cucumberautocomplete",
+		"wholroyd.HCL"
```

```suggestion
		"4ops.terraform"
```
For terraform I would prefer [`4ops.terraform`](https://marketplace.visualstudio.com/items?itemName=4ops.terraform) instead